CWE-94-Python_darwin/bakwc_PornDetector/pcr.py

```
@@ -0,0 +1,367 @@
+def predict():
+
+    positiveFiles, negativeFiles, testFiles = loadFileLists()
+    testFiles = testFiles
+
+    pcr = PCR()
+    pcr.loadModel('model.bin')
+    pred = pcr.predict(testFiles)
+    total = 0
+    correct = 0
+
+    for i in xrange(0, len(testFiles)):
```

Ruff (0.13.2)

Undefined name `xrange`

(F821)